### PR TITLE
refactor: bump jina version

### PR DESCRIPTION
**Pull request by florian-hoenicke**
the latest version has a fix that should allow us to run the executor tests together with all the other tests

### CODE REVIEW

**Comment by jupyterjazz** [comment]
Blocked because of a bug in a new core release. Waiting for core to release a hotfix.

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 pyfiglet==0.8.post1
-jina[perf]==3.12.0
+jina[perf]==3.13.2
 jina-hubble-sdk>=0.15.5
+finetuner==0.6.3
```
Comment (violenil) [inline requirements.txt:4]:
```suggestion

```

Comment (florian-hoenicke) [reply requirements.txt:4]: good suggestion thanks

**Comment by florian-hoenicke** [comment]
I open another pr that only bumps the version without something else

**Pull request closed without merging**